portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -51,8 +51,8 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-    int numberOfCommentsToDisplay = 0;
-    List<String> allTags = new ArrayList<>();
+  int numberOfCommentsToDisplay = 0;
+  List<String> allTags = new ArrayList<>();
 
     @Override
```

Comment:
The indentation for doGet() looks different than line 54-55. I think they should have 2-space indentation as line 54-55.